EPRN.PRNS.API/Controllers/PRNController.cs

```diff
@@ -164,6 +164,44 @@ public async Task<ActionResult> SaveDecemberWaste(int? id, bool decemberWaste)
             return Ok();
         }
 
+        [HttpGet]
+        [Route("Reference")]
+        public async Task<IActionResult> GetPrnReference(int id)
+        {
+            var prnDetailsDto = await _prnService.GetPrnDetails(id);
+            
+            if (prnDetailsDto == null)
```

Comment:
I wouldn't worry about that - the API has a facility for checking that the record exists. If the dto comes over as null, then we have different issues to it not being found